src/main/java/learn/petgallery/models/EntityEventType.java

```diff
@@ -0,0 +1,6 @@
+package learn.petgallery.models;
+
+public enum EntityEventType {
+    SAVE,
```

Comment:
Probably want ADD & UPDATE split out, but saving that for the "real" implementation if it comes to that.